help/sources/tutorials/ui/create/analytics/rainfocus-experience.md

```diff
@@ -0,0 +1,202 @@
+* [Create the Schema using the API](https://experienceleague.adobe.com/docs/experience-platform/xdm/tutorials/create-schema-api.html?lang=en)
+
+>[!IMPORTANT]
+>
+>* The schema must extend the **XDM ExperienceEvent class.** 
+> * You must ensure that the Schema includes a **primary identity**, and is **enabled for Profile**
+>   * [Define Identity Fields in the UI](https://experienceleague.adobe.com/docs/experience-platform/xdm/ui/fields/identity.html?lang=en)
+>* You may substitute the example identity (Email) for another appropriate identifier such as a sha256 email or ECID.
+
+## Enable the Integration Profile in RainFocus
+
+Now that the Service Account and XDM Schema are ready, switch to RainFocus to activate the Integration Profile, which will be responsible for streaming data to AEP. 
+
+1. Log into the **RainFocus** platform
+
+2. In the primary navigation, select **Libraries** and **Integration Profiles** 
+![Screenshot showing the RainFocus platform and the location of the Integration Profile in the main menu](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile.png)
+
+3. Select the **+** icon, in the bottom right corner, to create a new profile.
+
+4. Choose the **Adobe Experience Platform CDP** and click **OK**  
+![Screenshot showing the RainFocus platform and the location of the AEP CDP Integtaion Profile](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-select.png)
+
+5. Copy the following credential, previously created in the Adobe Developer Portal Project
+
+   1. **Client ID**
+   2. **Client Secret**
+   3. **Technical Account ID**
+   4. **Organization ID**
+
+6. Click **Save.** You should now see the new Integration Profile listed.
+
+7. Select the new Integration Profile and open it again.
+
+8. You will now see a list of predefined **push types** already configured. These are the [Experience Events](https://experienceleague.adobe.com/docs/experience-platform/xdm/classes/experienceevent.html?lang=en) that will be sent to AEP when they occur.   
+![Screenshot showing the RainFocus platform and the Experience Events that are configured automatically](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-setup.png)
+
+9. Select the **Sample JSON Payload** tab
+
+10. Highlight and copy the Sample JSON Payload and **save it in a new file with a .json extension**. We will use this in AEP to configure our mappings. 
+![Screenshot showing the RainFocus platform and the JSON preview that is generated](/help/sources/images/tutorials/create/rainfocus/rainfocus_integration-profile-json.png)
+
+> **Setup is not yet complete** - _we will need to return to complete the integrations profile, keep this tab open and proceed to the next step to install and configure the Source Connector in AEP and to obtain the_ **_Streaming Endpoint_** _and_ **_Dataflow ID._**
```

Comment:
- Let's turn this into a >[!TIP] styled note
- "AEP" -> "Experience Platform"
-